tools/scripts/publish.mjs

```diff
@@ -22,16 +23,20 @@ function invariant(condition, message) {
   }
 }
 
-// Executing publish script: node path/to/publish.mjs {name} --version {version} --tag {tag}
+// Executing publish script: node path/to/publish.mjs {name} --tag {tag}
 // Default "tag" to "next" so we won't publish the "latest" tag by accident.
-const [, , name, version, tag] = process.argv
+const [, , name, tag, otp] = process.argv
+
+// Fetch the version from "package.json" before publishing
+let version
+try {
+  const json = JSON.parse(readFileSync(`package.json`).toString())
+  version = json.version
+} catch (e) {
+  console.error(chalk.bold.red(`Error reading package.json file from library build output.`))
+  process.exit(1)
+}
```

Comment:
Changed the default publish command to use the package's version rathern than updating it based on cli param